recipe/meta.yaml

```diff
@@ -50,10 +50,6 @@ outputs:
   - name: libflang
     script: install_libflang.sh  # [unix]
     script: install_libflang.bat  # [win]
-    build:
-      track_features:
-        - emscripten
-      string: {{ variant }}_h{{ PKG_HASH }}_{{ build_number }} # [variant and variant.startswith("emscripten_")]
```

Comment:
If you want to not build one output, you'll have to add
```yaml
    build:
      string: true  # [variant and variant.startswith("emscripten_")]
```

Reply:
But I think it's going to be hard for flang to be usable without `libflang`, so lets see what the CI says here for now.

Reply:
`libflang` is only responsible for installing the *libFortranRuntime.\**, right?
So I'm following [Serge's instructions](https://github.com/serge-sans-paille/llvm-project/blob/feature/flang-wasm/README.wasm.md) and my plan for building this was to create an emscripten-forge recipe to build this runtime library with the `emsdk` setup that's already there to avoid conflicts with different emscripten versions.

Reply:
Regarding `libflang`, I think we could consider upgrading this recipe into a cross-compiler, at least per v19, where this gets a bit easier because we'll be dropping `libfortran-main`.

Reply:
How is this done? Are there any examples?

Reply:
This is honestly some of the trickiest stuff in conda-forge (normally we deal with maximum two architectures - build & {host,run}, but cross-compilers are essentially the only case where you need three), so I don't expect this for your contribution here.

That said, if you're curious and want dig into this a bit more, you can look at
* https://github.com/conda-forge/ctng-compilers-feedstock + https://github.com/conda-forge/ctng-compiler-activation-feedstock
* https://github.com/conda-forge/gfortran_impl_osx-64-feedstock + https://github.com/conda-forge/gfortran_osx-64-feedstock
* https://github.com/conda-forge/clang-compiler-activation-feedstock + https://github.com/conda-forge/clang-win-activation-feedstock + ...

When we switched the new flang here (see #28), we left out the cross-compilation capabilities at the time. But [most likely ](https://github.com/conda-forge/flang-feedstock/pull/28#issuecomment-1700182577)we'd have to create another feedstock for the activation, and leave this one to only do the implementation.